view/templates/admin/site.tpl

```diff
@@ -145,7 +145,7 @@
 	{{include file="field_input.tpl" field=$dbclean_expire_conv}}
 	<div class="submit"><input type="submit" name="page_site" value="{{$submit|escape:'html'}}" /></div>
 
-	<h3>{{$worker_title}}</h3>
+	<h3>{{$worker_title|escape}}</h3>
```

Comment:
What is your reasoning to escape translations?

Reply:
It doesn't hurt.

Reply:
Fair enough, if you like more work.

Reply:
In fact it had been less work. I used "grep" to search for some patterns that contain stuff that have to be escaped. These had been ```name}}``` and _drumroll_ ```title}```. So by escaping these ones, they didn't appear when I used "grep" again to search for more.